apis/io-engine/protobuf/v1/pool.proto

```diff
@@ -88,6 +91,22 @@ message ImportPoolRequest {
     EncryptionData data = 5;
     EncryptionSecret secret = 6;
   }
+  // The pool configuration specifying raid type and configuration if number of
+  // disks exceeds 1.
+  optional RaidConfig raid_config = 7;
+}
+
+// Pool raid configuration
+message RaidConfig {
+  oneof config {
+    Raid0Config raid0 = 1;
+  }
+}
+
+// RAID0 configuration parameters
+message Raid0Config {
+  // Strip size in KB (default 64)
```

Comment:
Any advantage to using KB instead of just bytes?

Reply:
In SPDK you configure 'strip_size_kb'. I used to use quantity type and bytes initially, but dealing with conversions it became a little confusing at some point. So to make it easier I opted to pass the value with the unit expected in io-engine/SPDK, while using Quantity-type in the CRD and ensure I have the conversion in one single place only.